Warnings and errors when running this example

I've cloned this repo and `npm install`ed. But when I run `karma start` I get following warnings:

```
WARN [preprocess]: Can not load "webpack", it is not registered!
  Perhaps you are missing some plugin?
WARN [preprocess]: Can not load "sourcemap", it is not registered!
  Perhaps you are missing some plugin?
INFO [karma]: Karma v0.12.37 server started at http://localhost:9876/
WARN [launcher]: Can not load "Chrome", it is not registered!
  Perhaps you are missing some plugin?
INFO [Chrome 43.0.2357 (Linux 0.0.0)]: Connected on socket MzTlihDi8ejzKVYkg9-3 with id manual-3092
```

Then when I open the URL from console I get this error:

```
Chrome 43.0.2357 (Linux 0.0.0) ERROR
  Uncaught ReferenceError: require is not defined
  at /home/janusz/devel/github/react-karma-webpack-testing/tests.webpack.js:1
Chrome 43.0.2357 (Linux 0.0.0): Executed 0 of 0 ERROR (0.203 secs / 0 secs)
```

What could be the reason? Am I missing any setup step?

Same problem with npm 2.1.3. Your dependencies are not in order in your package.json file.

```
INSERT  alex  ~  react-karma-webpack-testing  npm test                                                                                                                                                                                                                                                                                                      master

> react-karma-webpack-testing@1.0.0 test /Users/alex/react-karma-webpack-testing
> karma start

WARN [preprocess]: Can not load "webpack"!
  TypeError: Object [object Object] has no method 'refreshFiles'
    at Plugin.notifyKarmaAboutChanges (/Users/alex/react-karma-webpack-testing/node_modules/karma-webpack/index.js:108:15)
    at Plugin.<anonymous> (/Users/alex/react-karma-webpack-testing/node_modules/karma-webpack/index.js:72:9)
    at Tapable.applyPlugins (/Users/alex/react-karma-webpack-testing/node_modules/webpack/node_modules/tapable/lib/Tapable.js:26:37)
    at Watching._done (/Users/alex/react-karma-webpack-testing/node_modules/webpack/lib/Compiler.js:78:17)
    at Watching.<anonymous> (/Users/alex/react-karma-webpack-testing/node_modules/webpack/lib/Compiler.js:61:18)
    at Tapable.emitRecords (/Users/alex/react-karma-webpack-testing/node_modules/webpack/lib/Compiler.js:281:37)
    at Watching.<anonymous> (/Users/alex/react-karma-webpack-testing/node_modules/webpack/lib/Compiler.js:58:19)
    at /Users/alex/react-karma-webpack-testing/node_modules/webpack/lib/Compiler.js:274:11
    at Tapable.applyPluginsAsync (/Users/alex/react-karma-webpack-testing/node_modules/webpack/node_modules/tapable/lib/Tapable.js:60:69)
    at Tapable.afterEmit (/Users/alex/react-karma-webpack-testing/node_modules/webpack/lib/Compiler.js:271:8)
INFO [karma]: Karma v0.12.37 server started at http://localhost:9876/
INFO [launcher]: Starting browser Chrome
INFO [Chrome 44.0.2403 (Mac OS X 10.10.2)]: Connected on socket neccmsxR_HNA77OfQgsQ with id 15819656
Chrome 44.0.2403 (Mac OS X 10.10.2) ERROR
  Uncaught ReferenceError: require is not defined
  at /Users/alex/react-karma-webpack-testing/tests.webpack.js:1
Chrome 44.0.2403 (Mac OS X 10.10.2): Executed 0 of 0 ERROR (0.137 secs / 0 secs)
npm ERR! Test failed.  See above for more details.
 INSERT  alex  ~  react-karma-webpack-testing  npm -v                                                                                                                                                                                                                                                                                                        master
2.1.3
```

--- a/package.json
+++ b/package.json
@@ -9,18 +9,19 @@
   "author": "Justin Woo",
   "license": "MIT",
   "devDependencies": {
-    "babel-core": "^4.4.3",
-    "babel-loader": "^4.0.0",
+    "babel": "^5.6.14",
+    "babel-core": "^5.6.20",
+    "babel-loader": "^5.3.1",
     "expect": "^1.6.0",
-    "karma": "^0.12.31",
-    "karma-chrome-launcher": "^0.1.7",
+    "karma": "^0.13.3",
+    "karma-chrome-launcher": "^0.2.0",
     "karma-cli": "0.0.4",
     "karma-firefox-launcher": "^0.1.4",
     "karma-mocha": "^0.1.10",
-    "karma-sourcemap-loader": "^0.3.3",
-    "karma-webpack": "^1.5.0",
+    "karma-sourcemap-loader": "^0.3.5",
+    "karma-webpack": "^1.7.0",
     "mocha": "^2.1.0",
     "react": "^0.12.2",
-    "webpack": "^1.5.3"
+    "webpack": "^1.10.1"
   }
 }

